v3/index.html

```diff
@@ -0,0 +1,187 @@
+    <title>Daniil Dumshenko - CV</title>
+    <link rel="icon" href="cat.ico" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<header class="header">
+    <h1>Daniil Dumshenko</h1>
+    <img src="cat-profile-img.jpg" alt="Profile Image" class="profile-image">
+    <h2>Junior Software Engineer</h2>
+    <audio controls>
+        <source src="daniil-dumshenko.mp3" type="audio/mpeg">
+        Your browser does not support the audio element.
+    </audio>
+</header>
+
+<nav class="navbar">
+    <ul class="nav-ul">
+        <li class="nav-ul-item">
+            <a href="#contact" class="nav-ul-item-link">Contact</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#education" class="nav-ul-item-link">Education</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#about" class="nav-ul-item-link">About Me</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#experience" class="nav-ul-item-link">Work Experience</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#skills" class="nav-ul-item-link">Skills</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#projects" class="nav-ul-item-link">Projects</a>
+        </li>
+        <li class="nav-ul-item">
+            <a href="#languages" class="nav-ul-item-link">Languages</a>
+        </li>
+    </ul>
+</nav>
+
+<main class="main">
+    <section id="contact">
+        <h2>Contact</h2>
+        <ul>
+            <li>
+                <b>Phone:</b>
+                +49 176 87475645
+            </li>
+            <li>
+                <b>Email:</b>
+                <a href="mailto:daniil.dumshenko@gmail.com">
+                    daniil.dumshenko@gmail.com</a>
+            </li>
+            <li>
+                <b>LinkedIn:</b>
+                <a href="www.linkedin.com/in/laststap-undefined-24b554313">
+                    Profile</a>
+            </li>
+            <li>
+                <b>GitHub:</b>
+                <a href="https://github.com/LastStap">
+                    Profile</a>
+            </li>
+        </ul>
+    </section>
+
+    <section id="education">
+        <h2>Education</h2>
+        <table class="education-table">
+            <tbody>
+            <tr>
+                <th>Degree</th>
+                <th>Institution</th>
+                <th>Year</th>
+                <th>Grade</th>
+            </tr>
+            <tr>
+                <td>
+                    <b>Weiterbildung</b>
+                </td>
+                <td>Codillas Academy</td>
+                <td>2024 - 2025</td>
+                <td>?</td>
+            </tr>
+            <tr>
+                <td>
+                    <b>Bachelor of Computer Science</b>
+                </td>
+                <td>Cherkasy State Business College</td>
+                <td>2018 - 2022</td>
+                <td>1.6</td>
+            </tr>
+            </tbody>
```

Comment:
`<th>` має бути в `<thead>` а `<td>` в `<tbody>`
```suggestion
        <table class="education-table">
           <thead>
            <tr>
                <th>Degree</th>
                <th>Institution</th>
                <th>Year</th>
                <th>Grade</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>
                    <b>Weiterbildung</b>
                </td>
                <td>Codillas Academy</td>
                <td>2024 - 2025</td>
                <td>?</td>
            </tr>
            <tr>
                <td>
                    <b>Bachelor of Computer Science</b>
                </td>
                <td>Cherkasy State Business College</td>
                <td>2018 - 2022</td>
                <td>1.6</td>
            </tr>
            </tbody>
```